public/app/features/browse-dashboards/components/TagsCell.tsx

```diff
@@ -0,0 +1,27 @@
+import { css } from '@emotion/css';
+import React from 'react';
+import { CellProps } from 'react-table';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { TagList, useStyles2 } from '@grafana/ui';
+
+import { DashboardsTreeItem } from '../types';
+
+export function TagsCell({ row: { original: data } }: CellProps<DashboardsTreeItem, unknown>) {
+  const styles = useStyles2(getStyles);
+  const item = data.item;
+  if (item.kind === 'ui-empty-folder') {
+    return <></>;
+  }
+
+  return <TagList className={styles.tagList} tags={item.tags ?? []} />;
```

Comment:
similar question here, should we return an empty `TagList`, or only render `TagList` if tags exist? 🤔 e.g.

```suggestion
  return item.tags ? <TagList className={styles.tagList} tags={item.tags} /> : null;
```